internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -385,24 +409,43 @@ func (srv *Server) GetAllChangelogEntries(w http.ResponseWriter, r *http.Request
 	queryParams := r.URL.Query()
 	limit_param, ok := queryParams["limit"]
 
-	if !ok || len(limit_param) > 1{
-		paramError := errors.New("Invalid limit Parameter: does not exist or to many")
+	var needsLimit = true;
+	var needsOffset =  true;
+
+	if  len(limit_param) > 1{
+		paramError := errors.New("Invalid limit Parameter: Too many")
 		http.Error(w, paramError.Error(), http.StatusBadRequest)
 		return
 	}
 
+	if (!ok){
+		needsLimit = false
```

Comment:
`needsLimit` and `needsOffset` is confusing, plus we are checking the len of the parameters passed in twice, once on line 427 and again on 452, why not something like
```
if (len(offset_param) == 0 &&  len(limit_param) == 0){
    changelog, err = srv.DBClient.GetAllChangelogEntries()
} else {
    changelog, err = srv.DBClient.GetAllChangelogEntriesLimited(Offset,limit)
}
if err != nil {
    // handle err and return
}
// return changelog